src/blant-sampling.c

```diff
@@ -22,82 +22,96 @@ int *MCMCGetNeighbor(int *Xcurrent, GRAPH *G)
 {
     if (mcmc_d == 2)
     {
-	int oldu = Xcurrent[0];
-	int oldv = Xcurrent[1];
-	while (oldu == Xcurrent[0] && oldv == Xcurrent[1]) {
```

Comment:
It looks like more than 2/3 of the "changes" here are whitespace-only. Did you convert spaces to tabs, or vice versa? Or is it just indentation level? If it's the latter, then that's OK... otherwise I'd prefer to keep the whitespace the same.